pubtools/_quay/utils/misc.py

```diff
@@ -162,7 +165,8 @@ def setup_entry_point_cli(
             os.environ[key] = environ_vars[key]
         entry_point_func = pkg_resources.load_entry_point(*entry_tuple)
         if args:
-            func_args = [name]
+            if name:
+                func_args = [name]
             func_args.extend(args)
             yield functools.partial(entry_point_func, func_args)
```

Comment:
Not adding name is not enough to make pubtools-sign-cosign-signature-list work. I also changed this to:
```
            if entry_tuple[1] == "console_scripts":
                yield functools.partial(entry_point_func, func_args)
            else:
                yield functools.partial(entry_point_func, *func_args)
```

Reply:
ah yeah, makes sense